src/main/java/it/gov/pagopa/pu/processecexutions/repository/IngestionFlowFileRepository.java

```diff
@@ -35,4 +34,16 @@ public interface IngestionFlowFileRepository extends JpaRepository<IngestionFlow
     + "AND (:creationDate IS NULL OR iff.creationDate > :creationDate) "
     + "AND (:filename IS NULL OR iff.fileName = :fileName) ")
   Page<IngestionFlowFile> findByOrganizationIDFlowTypeCreateDate(Long organizationId, IngestionFlowFileType flowFileType, LocalDateTime creationDate, String fileName, Pageable pageable);
+
+  @Query("SELECT iff "
+    + "FROM IngestionFlowFile iff "
+    + "WHERE (:organizationId IS NULL OR iff.organizationId = :organizationId) "
+    + "AND (:flowFileType IS NULL OR iff.flowFileType = :flowFileType) "
+    + "AND (:fileName IS NULL OR iff.fileName ILIKE CONCAT('%', cast(:fileName as text), '%')) "
+    + "AND ((cast(:fromCreationDate as date) IS NULL AND cast(:toCreationDate as date) IS NULL) "
+      + "OR (cast(:fromCreationDate as date) IS NULL AND cast(:toCreationDate as date) IS NOT NULL AND iff.creationDate <= :toCreationDate) "
+      + "OR (cast(:toCreationDate as date) IS NULL AND cast(:fromCreationDate as date) IS NOT NULL AND iff.creationDate >= :fromCreationDate) "
+      + "OR (iff.creationDate BETWEEN :fromCreationDate AND :toCreationDate)) "
+    + "AND (:status IS NULL OR iff.status = :status) ")
+  Page<IngestionFlowFile> findIngestionFlowFiles(Long organizationId,IngestionFlowFileType flowFileType,String fileName,LocalDateTime fromCreationDate,LocalDateTime toCreationDate,IngestionFlowFileStatus status, Pageable pageable);
```

Comment:
instead of introduce a new query, let's modify the already existent `findByOrganizationIDFlowTypeCreateDate` method
(organizationId and flowFileType are mandatory filters)

instead of having all the OR condition to handle the interval filter let's handle separately from and to condition
... AND (:creationDateFrom IS NULL OR creationDate >= :...)
AND (:creationDateTo is NULL OR creationDate <=:...)

ps: let's handle the interval closed not opened
pps: who is actually using `findByOrganizationIDFlowTypeCreateDate` API? could we change `creationDate` param into `creationDateFrom`?